generators/client/templates/svelte/cypress/integration/account/change-password.spec.js.ejs

```diff
@@ -1,7 +1,7 @@
 describe('Change user password', () => {
 	beforeEach(() => {
 		cy.unregisterServiceWorkers()
-		cy.loginByApi('admin', 'admin')
+		cy.loginByApi(Cypress.env('adminUsername'), Cypress.env('adminPassword'))
```

Comment:
In the main generator, we allow you to set `CYPRESS_E2E_USERNAME` and `CYPRESS_E2E_PASSWORD` to override the default `admin` credentials. Does this support something similar?

Reply:
Yes, you can override the credentials with OS environment variables like `CYPRESS_adminUsername`, `CYPRESS_adminPassword` or directly under `cypress.json`.

Reply:
I think you should be consistent and use the same variable names that the main generator uses. AFAIK, the main generator never uses the `user` credentials to test anything.

Reply:
Here, I do have at least [one test](https://github.com/jhipster/generator-jhipster-svelte/runs/2823662541?check_suite_focus=true#step:20:476) to verify that authority as that's the out of box supported user/role. At moment, I believe that test is not included in `OIDC` flow due to a `skip-user-management` condition, however, I plan to fix that.